<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubgroupsForm_template</class>
 <widget class="QDialog" name="SubgroupsForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="The name of the dialog which lists the subgroups of students">Subgroups of students</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="yearTextLabel">
         <property name="text">
          <string>Year</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="yearsListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="groupsTextLabel">
         <property name="text">
          <string>Group</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="groupsListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="subgroupsTextLabel">
         <property name="text">
          <string>Subgroup</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="subgroupsListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget4">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="subgroupTextLabel">
         <property name="text">
          <string>Details</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="subgroupTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="addSubgroupPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifySubgroupPushButton">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeSubgroupPushButton">
       <property name="text">
        <string comment="It means &quot;Remove from here&quot; (but I want to keep the field shorter), remove the subgroup only from the current year/group. Please keep the translation short.">Remove here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="purgeSubgroupPushButton">
       <property name="text">
        <string comment="It means &quot;Remove from everywhere&quot; (but I want to keep the field shorter), remove the subgroup from all the years/groups in which it appears. Please keep the translation short.">Remove everywhere</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="activateStudentsPushButton">
     <property name="text">
      <string>Activate all activities for selected subgroup</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="deactivateStudentsPushButton">
     <property name="text">
      <string>Deactivate all activities for selected subgroup</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="commentsPushButton">
       <property name="text">
        <string>Comments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveSubgroupUpPushButton">
       <property name="text">
        <string comment="It means move the item up in the list">Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveSubgroupDownPushButton">
       <property name="text">
        <string comment="It means move the item down in the list">Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortSubgroupsPushButton">
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
